appengine/flexible_python37_and_earlier/extending_runtime/Dockerfile

```diff
@@ -0,0 +1,34 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# [START dockerfile]
+FROM gcr.io/google_appengine/python
+
+# Install the fortunes binary from the debian repositories.
+RUN apt-get update && apt-get install -y fortunes
+
+# Change the -p argument to use Python 2.7 if desired.
+RUN virtualenv /env -p python3.4
```

Comment:
Would it be possible to upgrade this? If not, fine to leave it as is for this PR's scope and update in a followup.

Reply:
Yes, will leave it to the future PRs.